k8s/consul/chart/consul/templates/application.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: "{{ .Release.Name }}"
   annotations:
     kubernetes-engine.cloud.google.com/icon: >-
-      data:image/png;base64, {{ .Files.Get "logo.png" | b64enc }} {{/* Get logo.png from the root of chart */}}
+      data:image/png;base64, {{ .Files.Get "logo.png" | b64enc }}
```

Comment:
Do we need to remove the space between `base64,` and the encoding as well?

```suggestion
      data:image/png;base64,{{ .Files.Get "logo.png" | b64enc }}
```

Reply:
Not sure if this space affects to something

Reply:
ok, tested, space actually breaks logo as well, going to check others